### PR TITLE
[21.09] Pin corresponding playbook repo to sync with Galaxy release

### PR DESCRIPTION
**Pull request by nuwang**
Pin the playbook branch so we can make sure it remains in sync with the Galaxy release.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Comment by nsoranzo** [comment]
@nuwang How should this be merged forward? Updated to 22.01 and 22.05 for `release_22.01` and `dev` respectively?
Also, if this needs to be kept in sync with the branch name, an item should be added to `RELEASE_ISSUE_TEMPLATE` in `scripts/bootstrap_history.py` .

File: .k8s_ci.Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ ARG STAGE1_BASE=python:3.7-slim
 ARG FINAL_STAGE_BASE=$STAGE1_BASE
 ARG GALAXY_USER=galaxy
 ARG GALAXY_PLAYBOOK_REPO=https://github.com/galaxyproject/galaxy-docker-k8s
+ARG GALAXY_PLAYBOOK_BRANCH=release_21.09
```
Comment (mvdbeek) [inline .k8s_ci.Dockerfile]: Could you read this from lib/galaxy/version.py ? Or possibly from a environment variables ?

**Comment by nuwang** [comment]
The playbook repo rarely changes, so my thinking was that we'd just keep it pinned to this release_21.09 branch indefinitely. Then, we only update the branch if a major/backward incompatible change occurs.

Perhaps alternatively, we could have a different versioning scheme for the playbook repo, just to make that clearer.

**Comment by nuwang** [comment]
I've made the relevant change. Hopefully, this makes it more maintainable?

**Review by afgane — APPROVED** [review]

**Pull request merged**